README.md

```diff
@@ -43,6 +43,7 @@ Thus, the project was named `Clair` after the French term which translates to *c
 [integrations]: /Documentation/integrations.md
 [CoreOS website]: https://coreos.com/clair/docs/latest/
 [Documentation directory]: /Documentation
+[Local Development]: /Documentation/local-development.md
```

Comment:
Add a bullet to the list that actually uses this link, haha

Reply:
done.